### ISSUE TITLE
cannot install without a git repository

### ISSUE THREAD

**Issue by jhgorse** [openafs-contrib/openafs-robotest#13]
No love for tarballs.
```
$ sudo ./install.sh
Installing afsutil
fatal: Not a git repository (or any of the parent directories): .git
```

I think we could leave this as wontfix so long as I can get to the githubs. =)

**Comment by meffie** [comment]
Oops, thanks for the report.

**Comment by meffie** [comment]
Fixed in commit 57a985a3cb660de150c5012172edd1b19d3f9271.

Thanks
